docs/src/modules/styles/getPageContext.js

```diff
@@ -4,14 +4,14 @@ import { create, SheetsRegistry } from 'jss';
 import rtl from 'jss-rtl';
 import { createMuiTheme, createGenerateClassName, jssPreset } from 'material-ui/styles';
 import blue from 'material-ui/colors/blue';
-import pink from 'material-ui/colors/pink';
+import orange from 'material-ui/colors/orange';
```

Comment:
Why not.
![capture d ecran 2018-01-09 a 08 02 10](https://user-images.githubusercontent.com/3165635/34708528-7db1b920-f513-11e7-8d12-0db087a451c0.png)

Reply:
Wait no, please revert, the contrast was 4.1 and it's 2.1 now.

Reply:
![capture d ecran 2018-01-09 a 08 42 36](https://user-images.githubusercontent.com/3165635/34709806-13d8e73e-f519-11e7-9dcc-1ae1d6165adf.png)
![capture d ecran 2018-01-09 a 08 42 24](https://user-images.githubusercontent.com/3165635/34709811-186cf178-f519-11e7-84c6-89f94928a89a.png)

Reply:
Dude, please read: https://github.com/mui-org/material-ui/pull/9794#issue-286903021